doc/how_to/server/reconnect.md

```diff
@@ -0,0 +1,106 @@
+# Re-connecting to a Session
+
+This guide walks you through configuring **automatic re-connection to Panel server sessions** when the connection is disrupted.
+
+---
+
+## Why re-connection matters
+
+Whenever a user visits an endpoint, a new server session is created.
+The session stays alive as long as the user remains connected.
+
+However, in some scenarios—such as **poor internet connectivity** or **proxies interrupting WebSocket connections**—the connection between the server and the frontend may close.
+
+By default, this **permanently disconnects** the application.
+Recent versions of Panel introduce the ability to **automatically re-connect** to the server instead of dropping the session.
+
+:::{note}
+This feature requires Panel **≥1.8**.
+If you are using an older version, you will need to upgrade.
+:::
+
+## Enabling re-connect
+
+To enable automatic server re-connection, set the corresponding configuration option:
+
+```python
+import panel as pn
+
+pn.config.reconnect = True
+
+# or equivalently
+pn.extension(reconnect=True)
```

Comment:
Do you also need to enable notifications for this to work? Like this `pn.extension(reconnect=True, notifications=True)`